cmd/mayactl/app/command/version.go

```diff
@@ -50,8 +53,67 @@ maya version
 			fmt.Println("m-apiserver status: ", mapiserver.GetConnectionStatus())
 
 			fmt.Println("Provider: ", orchprovider.DetectOrchProviderFromEnv())
+			util.CheckErr(checkLatestVersion(version.GetVersion()), util.Fatal)
 		},
 	}
 
 	return cmd
 }
+
+func checkLatestVersion(installedVersion string) error {
+	if installedVersion == "" {
+		return fmt.Errorf("GetVersion() returning empty string")
+	}
+
+	// removes first character i.e 'v' from the version
+	installedVersion = installedVersion[1:]
+
+	latestVersion, err := version.GetLatestRelease()
+	if err != nil {
+		return fmt.Errorf("found error - %s", err)
+	}
+
+	latest := parseVersion(latestVersion)
+	installed := parseVersion(installedVersion)
+
+	if latest == nil || installed == nil {
+		return fmt.Errorf("parseVersion() returning empty string")
+	}
+
+	flag := false
+
+	if installed[0] < latest[0] {
+		flag = true
+	} else if installed[1] < latest[1] && installed[0] == latest[0] {
+		flag = true
+	} else if installed[2] < latest[2] && installed[1] == latest[1] && installed[0] == latest[0] {
+		flag = true
+	}
+
+	if flag == true {
+		fmt.Println()
+		fmt.Println("A newer version of mayactl is available!")
+		fmt.Printf("Installed Version: v%s\n", installedVersion)
+		fmt.Printf("Latest version: v%s\n", latestVersion)
+	} else {
+		fmt.Println()
+		fmt.Println("You're running an up-to-date version of mayactl!")
+	}
```

Comment:
i think "else" is not required, we have to print the details if the new version is available.
@prateekpandey14 @kmova what do you think ?